### ISSUE TITLE
Question about FAWE API (undoing schematic paste on disable)

### ISSUE THREAD

**Issue by dieterblancke** [boy0001/FastAsyncWorldedit#1264]
Is there a way to undo an EditSession on server / plugin disable?

Currently I am using this (session being a field stored in the class):
```java
final EditSession editSession = WorldEdit.getInstance().getEditSessionFactory().getEditSession(session.getWorld(), -1, null, null);

session.undo(editSession);
session.flushQueue();
```

The problem with this is that the schematic paste doesn't get undone.

**Comment by didjee2** [comment]
Managed to fix the issue, I had to change this:

```java
final EditSession editSession = WorldEdit.getInstance().getEditSessionFactory().getEditSession(session.getWorld(), -1, null, null);

session.undo(editSession);
session.flushQueue();
```

to this:
```java
session.undo(session);
session.flushQueue();
```

Then none of the following bugs occured (these occured when using the method like in the documentation):
- undo during server runtime worked but relogging was required to see the change (or right clicking the blocks with hand)
- undo on server stop didn't work

Is this a mistake in the documentation perhaps?

**Comment by N0tMyFaultOG** [comment]
Your ticket has been closed because it falls into one of the following categories:

- Your issue has been fixed in a newer version of FastAsyncWorldEdit for versions 1.8 - 1.12.2, get the latest build [here](https://ci.athion.net/job/FastAsyncWorldEdit/lastSuccessfulBuild/artifact/target/) matching the platform you need.

- Your issue targets minecraft version 1.13 or higher. This is the wrong repository then. Move to [this repository](https://github.com/IntellectualSites/FastAsyncWorldEdit) and open a bug report here.
BUT BEWARE: You should consider updating to 1.15.2, using the latest Fawe build for 1.15.2 (Build 255 when writing this, get it [here](https://ci.athion.net/job/FastAsyncWorldEdit-1.15/)) and reproducing the issue in a clean environment (fresh setup Spigot or Paper server).

- Your issue has been fixed in a build of FAWE for 1.15 or an issue exists on our new repository.

- You either didn't fill out the template or deleted it.

If none of the bullet points suits, head to the second one.
